--- WString.cpp.orig	Tue Apr 16 16:07:30 2019
+++ WString.cpp	Sat Aug 24 01:04:56 2019
@@ -23,7 +23,7 @@
 
 #include "WString.h"
 #include "stdlib_noniso.h"
-#include "esp32-hal-log.h"
+// #include "esp32-hal-log.h"
 //extern "C" {
 //#include "esp_common.h"
 //}
@@ -182,7 +182,7 @@
         buffer = newbuffer;
         return 1;
     }
-    log_e("realloc failed! Buffer unchanged");
+    // log_e("realloc failed! Buffer unchanged");
     return 0;
 }
 
